src/ontology/mondo-edit.obo

```diff
@@ -430100,12 +430099,12 @@ xref: ICD10CM:G71.2 {source="Orphanet:363677", source="Orphanet:363677/attribute
 xref: MEDGEN:1673410 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: Orphanet:363677 {source="MONDO:equivalentTo"}
 xref: UMLS:C5192594 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1673410"}
-is_a: MONDO:0011577 ! myopathy, proximal, and ophthalmoplegia
+is_a: MONDO:0011577 {source="OMIM:605637"} ! myopathy, proximal, and ophthalmoplegia
```

Comment:
You cannot use OMIM:605637 as a source because OMIM:605637 is not equivalent to this term.

Reply:
I think you can use the orphanet ID here.